### PR TITLE
Add note about integrations to multiple clients js doc

### PR DESCRIPTION
**Pull request by AbhiPrasad**
## DESCRIBE YOUR PR

Copy over integrations snippet from https://docs.sentry.io/platforms/javascript/best-practices/browser-extensions/ to https://docs.sentry.io/platforms/javascript/best-practices/multiple-sentry-instances/ page.

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [ ] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [x] None: Not urgent, can wait up to 1 week+

## SLA

- Teamwork makes the dream work, so please add a reviewer to your PRs.
- Please give the docs team up to 1 week to review your PR unless you've added an urgent due date to it.
Thanks in advance for your help!

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## LEGAL BOILERPLATE

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## EXTRA RESOURCES

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

**Review by luc122c — CHANGES_REQUESTED** [review]
Don't forget to update the import!

```suggestion
-- defaultIntegrations
++ getDefaultIntegrations
```

**Review by mydea — APPROVED** [review]

**Review by s1gr1d — APPROVED** [review]
Looks good!

File: docs/platforms/javascript/common/best-practices/multiple-sentry-instances.mdx [hunk 1]
```diff
@@ -53,11 +55,23 @@ import {
...
+    ].includes(defaultIntegration.name);
+  },
+);
+
 const client = new BrowserClient({
   dsn: "___PUBLIC_DSN___",
   transport: makeFetchTransport,
   stackParser: defaultStackParser,
-  integrations: defaultIntegrations,
+  integrations: integrations,
```
Comment (s1gr1d) [inline docs/platforms/javascript/common/best-practices/multiple-sentry-instances.mdx:74]: Maybe you want to write it this way, but I leave it up to you
```suggestion
  integrations,
```

Comment (AbhiPrasad) [reply docs/platforms/javascript/common/best-practices/multiple-sentry-instances.mdx:74]: I thought about this, but this makes it especially explicit we are passing in this variable here. I'll leave this syntax for people who are less familiar with JS.

**Pull request merged**